src/btree/bt_stat.c

```diff
@@ -242,7 +242,7 @@ __stat_page_col_var(WT_SESSION_IMPL *session, WT_PAGE *page, WT_DSRC_STATS **sta
     }
 
     /* Walk any append list. */
-    WT_SKIP_FOREACH (ins, WT_COL_APPEND(page))
+    WT_SKIP_FOREACH (ins, WT_COL_APPEND(page), WT_ATOMIC_RELAXED)
```

Comment:
We need to use a relaxed read for shared variables even though they are not used in the shared context, otherwise the compiler will by default using the strictest order and will introduce unnecessary overhead. Fortunately, clang helps us find all these cases.